src/index.js

```diff
@@ -1,12 +1,22 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import * as Sentry from "@sentry/react";
+import { Integrations } from "@sentry/tracing";
```

Comment:
@eldemcan shall we import these dynamically? So if integration is not enabled we don't include this module in the bundle? 

What I mean is something like: 

```
if (process.env.SENTRY) { 
  import("@sentry/react").then(Sentry => {
    // ... here goes the rest 
  })
}
```

Reply:
@svedova fantastic idea, I wasn't aware of this. Since we need to import two modules do we need to do a proxy module? 

Do I need to create something like errormonitoring.js which imports

```
import * as Sentry from "@sentry/react";
import { Integrations } from "@sentry/tracing";
```


Then I will try importing that with dynamic import?

Reply:
We could also use an anonymous function and perform the imports inside of it:

```
(async () => {
  const Sentry = await import('@sentry/react');
  const { Integrations } = await import('@sentry/tracing');
})()
```

This way we could avoid creating one more file 👍